helios-tools/pom.xml

```diff
@@ -58,6 +58,10 @@
       <groupId>com.typesafe</groupId>
       <artifactId>config</artifactId>
     </dependency>
+    <dependency>
+      <groupId>javax.ws.rs</groupId>
+      <artifactId>jsr311-api</artifactId>
```

Comment:
or `javax.ws.rs:javax.ws.rs-api` 

there seems to be 1000 variations of that API jar